tests/integration/test_validate_synced_files_existance.py

```diff
@@ -0,0 +1,98 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from pathlib import Path
+
+import pytest
+
+from packit.api import PackitAPI
+from packit.utils.commands import cwd
+
+
+@pytest.mark.parametrize(
+    "raw_package_config,expected_output",
+    [
+        (
+            """
+            {
+                "config_file_path": "packit.json",
+                "dist_git_base_url": "https://packit.dev/",
+                "downstream_package_name": "packit",
+                "upstream_ref": "last_commit",
+                "upstream_package_name": "packit_upstream",
+                "create_tarball_command": ["commands"],
+                "allowed_gpg_keys": ["gpg"],
+                "dist_git_namespace": "awesome",
+                "synced_files": [{ "src": "a.md", "dest": "aaa.md" }, "a_dir"]
+            }
+            """,
+            "packit.json is valid and ready to be used",
+        ),
+        (
+            """
+            {
+                "config_file_path": "packit.json",
+                "dist_git_base_url": "https://packit.dev/",
+                "downstream_package_name": "packit",
+                "upstream_ref": "last_commit",
+                "upstream_package_name": "packit_upstream",
+                "create_tarball_command": ["commands"],
+                "allowed_gpg_keys": ["gpg"],
+                "dist_git_namespace": "awesome",
+                "synced_files": ["a.md", "b.md", "c.txt", "a_dir"]
+            }
+            """,
+            "The following path does not exist: c.txt",
+        ),
+    ],
+    ids=[
+        "none_missing",
+        "one_missing",
+    ],
+)
+def test_validate_synced_files_one_missing(tmpdir, raw_package_config, expected_output):
```

Comment:
I wanted to suggest one example with all paths correct (=existing), but now I see we have one. Nice. Can you please just rename the test method to something more general so it fits both use cases?

Reply:
And please also rename the file to `test_validate_synced_files.py` in order to keep it simple :-)